api4/config_test.go

```diff
@@ -470,6 +475,10 @@ func TestUpdateConfigDiffInAuditRecord(t *testing.T) {
 	})
 	require.Equal(t, timeoutVal+1, *cfg.ServiceSettings.ReadTimeout)
 
+	// Shutting down the server to make sure all logged records get flushed.
+	th.TearDown()
```

Comment:
Here you could just call `th.Server.Log.Flush()`.  I suggest doing that anyway in case the "flush not called on shutdown" issue is still in this version of server (I suspect it is).

Shutdown does make attempts to drain any queues but is not as complete as flush.

Reply:
Awesome, will update :+1: